internal/protoschema/plugin/pluginbigquery/pluginbigquery.go

```diff
@@ -47,8 +56,10 @@ func Handle(
 			if len(data) == 0 || string(data) == "null" {
 				continue
 			}
+			name := tableName + "." + bigquery.FileExtension
```

Comment:
For this to truly be a drop-in replacement for `bq-schema` we'd need to make the suffix customizable or just use whatever they do. I.e., we have it as `*.TableSchema.json` whereas they use `*.schema`.

Reply:
FWIW, I'd prefer a `.json` extension, even if just for file formatting. Calling it `.schema` doesn't add much clarity IMO, and was strange when I started using their generator. Not a big deal either way.

Reply:
Thanks for your feedback @derekperkins. 

We've been chatting internally about this plugin and whether we want to support the existing custom options from bq_schema, create our own or support both for compatibility.

Likewise, there's the decision of where to output the files, because we want users to have a consistent experience using the plugins in this repository.

TL;DR - the plugins are still fairly alpha and we plan to revisit some specific behavior in the next few weeks. But wanted to give you an update in case you were wondering what's up with this plugin.

Reply:
No problem. `protoc-gen-bq-schema` is working just fine for me at the moment. I'm a big buf fan, so I was happy to test it out. My question in #6 was to understand if there was a specific end goal different for this plugin, or if it was just a cleaner, more reliable implementation. I'm more than happy for you to take your time architecting, and will give feedback along the way